src/gl/textureStreamAV.cpp

```diff
@@ -106,7 +109,7 @@ bool TextureStreamAV::load(const std::string& _path, bool _vFlip, TextureFilter
         // av_dict_set(&options, "framerate", toString(getFps()).c_str(), 0);
         // std::cout << "Opening " << driver << " at " << _path << std::endl;
 
-        AVInputFormat *ifmt = av_find_input_format(driver.c_str());
+        const AVInputFormat *ifmt = av_find_input_format(driver.c_str());
```

Comment:
Trying this on my Ubuntu 20.04 and it doesn't compile as `const`. Did you added it because it's required on newer versions?